templates/documentation/beyond-canonical.html

```diff
@@ -14,12 +14,12 @@
 <div class="p-strip u-no-padding--top">
   <div class="row" id="documentation">
     <aside class="col-3">
-      <div class="p-side-navigation--raw-html is-sticky" id="drawer" style="top: 4.5rem; z-index: 37;">
+      <div class="p-side-navigation--raw-html is-sticky" id="drawer">
```

Comment:
Not entirely sure if this is the reason, but now the first element of the navigation is "overscrolling".


https://user-images.githubusercontent.com/14939793/220315502-63f60f25-b170-4161-999e-98048a119bcb.mov

Reply:
Good catch. I have updated my PR, could you take another look please